include/OpenSpaceToolkit/Astrodynamics/Flight/System/Dynamics/CentralBodyGravity.hpp

```diff
@@ -104,6 +113,8 @@ class CentralBodyGravity : public Dynamics
 
    private:
     Shared<const Celestial> celestialObjectSPtr_;
+    Integer positionIndex_ = Integer::Undefined();
+    Integer velocityIndex_ = Integer::Undefined();
```

Comment:
I would make it even more explicit with `positionCoordinatesIndex_` and `velocityCoordinatesIndex_`, what do you think?